rmgpy/rmg/reactors.py

```diff
@@ -690,6 +691,28 @@ def to_rms(obj, species_names=None, rms_species_list=None, rmg_species=None):
                 # means it is fragment's cutting label
                 pass
         bondnum = len(obj.molecule[0].get_all_edges())
+        
+        if isinstance(obj.molecule[0], Fragment):
+            th = obj.get_thermo_data()
+            thermo = to_rms(th)
+            return rms.Species(
+                name=obj.label,
+                index=obj.index,
+                inchi="",
+                smiles="",
+                adjlist="",
+                thermo=thermo,
+                atomnums=atomnums,
+                bondnum=bondnum,
+                diffusion=rms.EmptyDiffusivity(),
+                radius=0.0,
+                radicalelectrons=obj.molecule[0].multiplicity - 1,
+                molecularweight=0.0,
+                henrylawconstant=rms.EmptyHenryLawConstant(),
+                liquidvolumetricmasstransfercoefficient=rms.EmptyLiquidVolumetricMassTransferCoefficient(),
+                comment=obj.thermo.comment,
+            )
+        
```

Comment:
I don't think we should hard code that fragment does not have diffusion, have no molecular weights, and have no kLA and kH. We should reuse what we have below for molecule, but change the following parts
- estimates of the species radius for fragments should be done with the representative molecule
- use this estimate of radius from the representative molecule to estimate stokes diffusivity, instead of hard coded no diffusion limitation
- same goes for kLA and kH for fragments. Fragments' solvation free energy should be estimated using their representative molecule.

Reply:
I can push some changes regarding this because this falls into what I have been helping Yen-Ting with over the past year if you're fine with it

Reply:
Please do, thank you.